index.js

```diff
@@ -37,7 +37,8 @@ module.exports = function (PIXI)
 
         var loadOptions = {
             crossOrigin: resource.crossOrigin,
-            loadType: Resource.LOAD_TYPE.IMAGE
+            loadType: Resource.LOAD_TYPE.IMAGE,
+            parentResource: resource
```

Comment:
can't `add` anything to the loader while it is running unless it has a parentResource, which is the case for the spritesheets.